src/ansys/tools/path/path.py

```diff
@@ -1088,57 +1088,49 @@ def get_mechanical_path(
     return _get_application_path("mechanical", allow_input, version, find)
 
 
-def _mechanical_version_from_path(path: str) -> int:
-    """Extract the Ansys Mechanical version from a path.
-
-    Generally, the version of Mechanical is contained in the path:
-
-    - On Windows, for example: ``C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe``
-    - On Linux, for example: ``/usr/ansys_inc/v231/aisol/.workbench``
+def _version_from_path(path: str, product_name: str, path_version_regex: str) -> int:
+    """Extract the version from the executable path.
 
     Parameters
     ----------
-    path : str
-        Path to the Mechanical executable file.
+    path: str
```

Comment:
Since this is a internal function , I think we should update the doc string on `version_from_path` function rather than here.